RecoTracker/IterativeTracking/python/iterativeTkConfig.py

```diff
@@ -133,8 +157,12 @@ def _seedOrTrackProducers(era, typ):
         else:
             ret.append(seeder)
 
-    for i in _iterations_muonSeeded:
-        ret.append(_modulePrefix(i).replace("Step", typ))
+    if era == "trackingPhase2PU140":
+        for i in _iterations_muonSeeded_trackingPhase2PU140:
+            ret.append(_modulePrefix(i).replace("Step", typ))
+    else :
+        for i in _iterations_muonSeeded:
+            ret.append(_modulePrefix(i).replace("Step", typ))
```

Comment:
Something like following would be more generic
```
for i in globals().get("_iterations_muonSeeded"+postfix(era), _iterations_muonSeeded)
    ret.append(_modulePrefix(i).replace("Step", typ))
```
(essentially the same comment as https://github.com/makortel/cmssw/pull/2/files#r69396672)